client/collections/articles.coffee

```diff
@@ -18,8 +18,10 @@ module.exports = class Articles extends Backbone.Collection
       imgSrc: article.get('thumbnail_image')
       title: article.get('thumbnail_title')
       subtitle: (
-        if article.get('published_at')
+        if article.get('published_at') && article.get('published')
           "Published #{moment(article.get('published_at')).fromNow()}"
+        else if article.get('scheduled_publish_at')
+          "*****Scheduled to be published #{moment(article.get('scheduled_publish_at')).fromNow()}*****"
```

Comment:
That's an aggressive amount of asterisks :-P—I think we could probably ditch the asterisks in favor of a color e.g.

`<span class='article-list-scheduled'>Scheduled to be published #{moment(article.get('scheduled_publish_at')).fromNow()}</span>`